.github/workflows/dotnet-tests.yml

```diff
@@ -0,0 +1,64 @@
+name: Dotnet Framework Tests
+on: [push, pull_request]
+
+jobs:
+  run-babelfish-dotnet-tests:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        id: checkout
+
+      - name: Install Dependencies
+        id: install-dependencies
+        if: always()
+        uses: ./.github/composite-actions/install-dependencies
+
+      - name: Build Modified Postgres
+        id: build-modified-postgres
+        if: always() && steps.install-dependencies.outcome == 'success'
+        uses: ./.github/composite-actions/build-modified-postgres
+      
+      - name: Compile ANTLR
+        id: compile-antlr
+        if: always() && steps.build-modified-postgres.outcome == 'success'
+        uses: ./.github/composite-actions/compile-antlr
+      
+      - name: Build Extensions
+        id: build-extensions
+        if: always() && steps.compile-antlr.outcome == 'success'
+        uses: ./.github/composite-actions/build-extensions
+      
+      - name: Install Extensions
+        id: install-extensions
+        if: always() && steps.build-extensions.outcome == 'success'
+        uses: ./.github/composite-actions/install-extensions
+      
+      - name: Install Dotnet
+        id: install-dotnet
+        if: always() && steps.install-extensions.outcome == 'success'
+        run: | 
+          cd ~
+          wget https://packages.microsoft.com/config/ubuntu/20.04/packages-microsoft-prod.deb -O packages-microsoft-prod.deb
```

Comment:
This might not be necessary since we are grabbing prod.list in the install-dependencies workflow, but I'm not sure, I don't understand these microsoft packages that well. Also, we can probably remove all these sudo apt-get update commands now, since its already being performed in the install-dependencies workflow

Reply:
prod.list is just a list of package locations, .deb contains the actual files, i believe this should remain a .net specific command.